keep/providers/newrelic_provider/newrelic_provider.py

```diff
@@ -13,15 +14,22 @@
 from keep.exceptions.provider_config_exception import ProviderConfigException
 from keep.exceptions.provider_exception import ProviderException
 from keep.providers.base.base_provider import BaseProvider
-from keep.providers.models.provider_config import ProviderConfig
+from keep.providers.models.provider_config import ProviderConfig, ProviderScope
+from pybars import Compiler
```

Comment:
What is this?

Reply:
New Relic uses handlebars syntax, I was not having any luck with string formatting. pybars is package to support the same.

Let me make a commit where I remove all uneccessary code. 

Thanks for pointing that @talboren.